incubator-tools/documentai_migrating_schema_between_processors/Document AI Migrating Schema Between Processors.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>This should really be an official code sample embedded in the documentation. Maybe on this page or somewhere similar? <a href="https://cloud.google.com/document-ai/docs/workbench/train-processor" target="_blank">https://cloud.google.com/document-ai/docs/workbench/train-processor</a></p><p>But lets leave as is for now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>